src/command/cmd_funcs.c

```diff
@@ -7117,7 +7117,8 @@ gboolean
 cmd_plugins_uninstall(ProfWin* window, const char* const command, gchar** args)
 {
     if (args[1] == NULL) {
-        return FALSE;
+        cons_show("Please specify plugin name, see /help plugins");
```

Comment:
We usually use `cons_bad_cmd_usage(command);` for this. See other functions as an example.

Reply:
fixed